### PR TITLE
Fix docs workflow

### PR DESCRIPTION
**Pull request by mdellweg**
Python version 3.11 seems to be required.

[noissue]

### CODE REVIEW

File: .github/workflows/all-docs.yml [hunk 1]
```diff
@@ -1,42 +1,40 @@
 ---
-name: Publish staging-docs.pulpproject.org
+name: "Publish docs.pulpproject.org"
 on:
   schedule:
-    # * is a special character in YAML so you have to quote this string
-    # runs at 3:00 UTC daily on weekdays 1 to 5 (no weekends)
```
Comment (mdellweg) [inline .github/workflows/all-docs.yml:6]: One should quote all strings if one was forced to use yaml.

File: .github/workflows/scripts/publish_all_docs.sh [hunk 2]
```diff
@@ -20,9 +20,9 @@ export WORKSPACE=$PWD
...
 
-mkdir -p ../staging-docs.pulpproject.org
-tar -xvf staging-docs.pulpproject.org.tar --directory ../staging-docs.pulpproject.org
-pushd ../staging-docs.pulpproject.org
+mkdir -p ../docs.pulpproject.org
+tar -xvf docs.pulpproject.org.tar --directory ../docs.pulpproject.org
+pushd ../docs.pulpproject.org
 
 # publish to pulpproject.org
 rsync -avzh --delete site/ doc_builder_staging_pulp_core@docs.pulpproject.org:/var/www/docs.pulpproject.org/staging_pulp_core/
```
Comment (mdellweg) [inline .github/workflows/scripts/publish_all_docs.sh:28]: Just removing the "staging" from this line is probably not going to work, right?

**Review by pedro-psb — CHANGES_REQUESTED** [review]
The current domain doesnt have the `docs.` prefix.
s/docs.pulpproject.org/pulpproject.org

File: .github/workflows/scripts/publish_all_docs.sh (hunk 2 above)
Comment (mdellweg) [reply .github/workflows/scripts/publish_all_docs.sh:28]: Fixing the wf is more important atm.

**Review by pedro-psb — APPROVED** [review]

**Review by ggainey — APPROVED** [review]

**Review by lubosmj — APPROVED** [review]
Formally approving.

**Pull request merged**